### PR TITLE
creating a maintainers file

### PR DESCRIPTION
**Pull request by miao0miao**
This file contains references to other repositories, they will include references to this file.

### CODE REVIEW

**Review by t-lo — CHANGES_REQUESTED** [review]
Can re rename `maintainers` -> `Maintainers.md` please?

**Comment by sayanchowdhury** [comment]
> Can re rename `maintainers` -> `Maintainers.md` please?

Could it be MAINTAINERS.md or MAINTAINERS? Following the convention of README.md

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* Thilo Fromm @t-lo
+
+### scripts
+maintainers:
+* Kai Lüke @pothos
+* Thilo Fromm @t-lo
+
+### coreos-overlay
+maintainers:
+* Krzesimir Nowak @Krzesimir Nowak @krnowak
```
Comment (tormath1) [inline MAINTAINERS.md]: Any reason to have duplicated name?
```suggestion
* Krzesimir Nowak @krnowak
```

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+maintainers:
+* Mathieu Tortuyaux @tormath1
+
+### baselayout
+maintainers:
+* Kai Lüke @pothos
+
+### sysext-bakery
+maintainers:
+Kai Lüke @pothos
```
Comment (tormath1) [inline MAINTAINERS.md]: For consistency: 
```suggestion
* Kai Lüke @pothos
```

File: MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+maintainers:
+* Mathieu Tortuyaux @tormath1
+
+### locksmith
+maintainers:
+* Mathieu Tortuyaux @tormath1
+
+### update_engine
+maintainers:
+* wrl
```
Comment (tormath1) [inline MAINTAINERS.md]:
```suggestion
* @wrl
```

File: MAINTAINERS.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+* Thilo Fromm @t-lo
+
+### flatcar-docs
+maintainers:
+* Kai Lüke @pothos
+* Thilo Fromm @t-lo
+
+### portage-stable
+maintainers:
+* Krzesimir Nowak @Krzesimir Nowak @krnowak
```
Comment (tormath1) [inline MAINTAINERS.md]: ditto.

File: MAINTAINERS.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+maintainers:
+* wrl
+
+### FLUO 
+maintainers:
+* Mateusz Gozdek @invidian
+
+### init
+maintainers:
+* init
```
Comment (tormath1) [inline MAINTAINERS.md]: I don't think there is yet a primary maintainer for this? Maybe @pothos ?

File: MAINTAINERS.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+## Repositories
+
+### Flatcar 
+maintainers:
+* Thilo Fromm @t-lo
+
+### scripts
+maintainers:
+* Kai Lüke @pothos
```
Comment (t-lo) [inline MAINTAINERS.md:14]:
```suggestion
* Kai Lüke @pothos
* Gabriel Samfira @gabriel-samfira
```

**Review by t-lo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**